packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -163,10 +163,23 @@
 		}
 	}
 
-	.block-editor-block-mover:not(.is-horizontal) .block-editor-block-mover__move-button-container {
+	.block-editor-block-mover .block-editor-block-mover__move-button-container {
 		width: auto;
 	}
 
+	.block-editor-block-mover.is-horizontal {
+		.block-editor-block-mover__move-button-container,
+		.block-editor-block-mover-button {
+			padding-left: 6px;
```

Comment:
Should we use `$grid-unit` here?

Reply:
There isn't a standard `$grid-unit` for `6px`.

I created a new `$grid-unit-075` locally to this stylesheet to represent `6px` being `0.75 x 8px`.

Reply:
👍 Not a problem. 

I wasn't very keen on having `$grid-unit-075` there so kept those changes in a single easy to revert commit.

That revert has been done in https://github.com/WordPress/gutenberg/pull/49747/commits/32b948ccfb4b377f9bbbbe9d54f95662394d2d6c